### PR TITLE
changed unit for fiber_photometry_response_series to n.a.

### LINKED ISSUE #20: `FiberPhotometryResponseSeries` unit should be "n.a."
This should be corrected in the example as well:
https://github.com/catalystneuro/ndx-fiber-photometry/blob/94d80c2996cf3c71bd7e6f2b2c83f9b3fb9906ff/src/pynwb/tests/integration/test_fiber_photometry.py#L203

Maybe we could "hardcode" it into the schema as well so the default is always "n.a." to avoid mistakes? @alessandratrapani

### PR DESCRIPTION
**Pull request by pauladkisson**
Fixes #20

### CODE REVIEW

**Comment by pauladkisson** [comment]
@weiglszonja, what is the point of specifying a default value for unit if there is an error anytime you don't specify it?

**Comment by weiglszonja** [comment]
Well I thought we can set a default value without having to specify every time

File: spec/ndx-fiber-photometry.extensions.yaml [hunk 1]
```diff
@@ -263,6 +263,11 @@ groups:
       - null
     doc: The data values. May be 1D or 2D. The first dimension must be time.The optional
       second dimension refers to the fiber that record the series.
+    attributes:
+    - name: unit
+      dtype: text
+      value: n.a.
```
Comment (weiglszonja) [inline spec/ndx-fiber-photometry.extensions.yaml:269]: How about `default_value` as in https://github.com/NeurodataWithoutBorders/nwb-schema/blob/d8d8e0332e72cd14b10bd7e809558eca31dad391/core/nwb.base.yaml#L140
```suggestion
      value: n.a.
      default_value: n.a.
```

**Comment by pauladkisson** [comment]
OK, so the problem with specifying a data unit for times series objects isn't with `value` or `default_value`, the problem is that `TimeSeries` init requires users to specify unit, and the value for unit passed into the TimeSeries __init__ is always what ends up in the nwb file.

We could probably change this behavior by redefining __init__ for FiberPhotometryResponseSeries and CommandedVoltageSeries, but I think it's better to leave the time series alone and just remove the redundant/misleading data-unit attributes from our spec.

What do you think @weiglszonja?

**Comment by weiglszonja** [comment]
> OK, so the problem with specifying a data unit for times series objects isn't with `value` or `default_value`, the problem is that `TimeSeries` init requires users to specify unit, and the value for unit passed into the TimeSeries **init** is always what ends up in the nwb file.
> 
> We could probably change this behavior by redefining **init** for FiberPhotometryResponseSeries and CommandedVoltageSeries, but I think it's better to leave the time series alone and just remove the redundant/misleading data-unit attributes from our spec.
> 
> What do you think @weiglszonja?

I see, thanks for looking into this. I think that’s a good idea

**Pull request merged**